README.md

```diff
@@ -93,380 +93,54 @@
-
-    def _forward(self, x):
-        x = self.linear0(x)
-        x = ivy.functional.relu(x)
-        x = self.linear1(x)
-        return x
-```
-
-If we put it all together, we\'ll have something like this. This example
-uses PyTorch as the backend, but this can easily be changed to your
-favorite frameworks, such as TensorFlow, or JAX.
-
-``` python
-import ivy
-
-
-class Regressor(ivy.Module):
-    def __init__(self, input_dim, output_dim):
-        self.input_dim = input_dim
-        self.output_dim = output_dim
-        super().__init__()
-
-    def _build(self, *args, **kwargs):
-        self.linear0 = ivy.Linear(self.input_dim, 128)
-        self.linear1 = ivy.Linear(128, self.output_dim)
-
-    def _forward(self, x):
-        x = self.linear0(x)
-        x = ivy.functional.relu(x)
-        x = self.linear1(x)
-        return x
-
-ivy.set_backend('torch')  # set backend to PyTorch (or any other backend!)
-
-model = Regressor(input_dim=1, output_dim=1)
-optimizer = ivy.Adam(0.3)
-
-n_training_examples = 2000
-noise = ivy.random.random_normal(shape=(n_training_examples, 1), mean=0, std=0.1)
-x = ivy.linspace(-6, 3, n_training_examples).reshape((n_training_examples, 1))
-y = 0.2 * x ** 2 + 0.5 * x + 0.1 + noise
-
-
-def loss_fn(v, x, target):
-    pred = model(x, v=v)
-    return ivy.mean((pred - target) ** 2)
-
-for epoch in range(40):
-    # forward pass
-    pred = model(x)
-
-    # compute loss and gradients
-    loss, grads = ivy.execute_with_gradients(lambda params: loss_fn(*params), (model.v, x, y))
-
-    # update parameters
-    model.v = optimizer.step(model.v, grads)
-
-    # print current loss
-    print(f'Epoch: {epoch + 1:2d} --- Loss: {ivy.to_numpy(loss).item():.5f}')
-
-print('Finished training!')
-```
-
-The model\'s output can be visualized as follows:
-
-<div align="center">
-   <img width="50%" class="dark-light" src="https://raw.githubusercontent.com/unifyai/unifyai.github.io/main/img/regressor_lq.gif">
-</div>
-
-
-As always, you can find more information about [Ivy as a framework in
-the
-docs!](https://unify.ai/docs/ivy/overview/design/ivy_as_a_framework.html)
+# Getting started: 
 
-### When should I use Ivy as a framework?
+You can start by reading tutorials and explore our demos, 
+Here is a good starting point for you:
+- [How to Convert your code?](https://unify.ai/docs/ivy/demos/learn_the_basics/04_transpile_code.html) 
+- [How to write a framework-free code?](https://unify.ai/docs/ivy/demos/learn_the_basics/02_unify_code.html) 
+- Accelerate your development (WIP)
+- Autotune and optimize models (WIP) 
 
-As Ivy supports multiple backends, writing code in Ivy breaks you free
-from framework limitations. If you want to publish highly flexible code
-for everyone to use, independently of the framework they are using, or
-you plan to develop ML-related tools and want them to be interoperable
-with not only the already existing frameworks, but also with future
-frameworks, then Ivy is for you!
-
-## Setting up Ivy
```

Comment:
We should definitely keep the instructions to install ivy in the readme, if someone goes to the repo and just want to install ivy they shouldn't have to look for the instructions in the docs, this should almost be a copy of https://unify.ai/docs/ivy/overview/get_started.html imo

Reply:
we can reference some sections but the basic installation instructions should definitely be here